sway-fmt-v2/src/fmt.rs

```diff
@@ -93,8 +106,9 @@ enum Color {
                     Grey: (), }
         "#;
 
-        // Until #1995 is addressed we will not have contract; in the output
-        let correct_sway_code = r#"enum Color {
+        let correct_sway_code = r#"contract;
```

Comment:
I removed the separate test which was not testing things correctly (even though I remove the changes from this PR, it wouldn't fail as the failure depends on the presence of other items in the source code that we are trying to format). Although we do not have any separate test for program type, it is checked in each test.